.github/scripts/acquire-build-image

```diff
@@ -101,17 +106,19 @@ class Shell:
         print(stderr)
         print("-------------------")
 
-        not_found_messages = ["not found: manifest unknown"]
-        throttle_messages = ["toomanyrequests: Rate exceeded", "toomanyrequests: Data limit exceeded"]
```

Comment:
Looks like this was unintentionally reverted. We definitely want this "Data limit exceeded" retry for forked PRs.

Reply:
oh thanks I saw that and TODO but forgot